yellowbrick/classifier/prcurve.py

```diff
@@ -173,7 +173,8 @@ def fit(self, X, y=None):
             self.estimator = OneVsRestClassifier(self.estimator)
 
             # Use label_binarize to create multi-label ouptut for OneVsRestClassifier
-            Y = label_binarize(y, classes=np.unique(y))
+            self.classes_from_y = np.unique(y)
```

Comment:
This is a good solution, thank you! In order to remove confusion with `classes_`, might I propose that we make this `_classes_from_y` or `_unique_y` or `_target_labels`?

Reply:
Sure.